app/lib/liquid/template_support.rb

```diff
@@ -48,7 +48,21 @@ def add_liquid_view_paths(options = {})
         end
       end
 
-      view_renderer.extend(LayoutSupport)
+      ActionView::Renderer.class_eval do
+        # RAILS: this overrides a Rails method. Beware when upgrading.
+        # We overwrite ActionView methods to make Liquid work. Due to changes in ActionView, we cannot depend on
+        # overwriting instances of ActionView::Renderer anymore, as those can be reinitialized during a page render.
+        # Because of that, we need to overwrite this method in class level, which may cause odd behaviours without
+        # the respond_to?(:liquify) check.
+        # Maybe move this to an initializer in case we can't find a better solution?
+        def render_template_to_object(context, options)
```

Comment:
I don't really understand this change :upside_down_face: but it seems that it all works :grimacing:

Reply:
I've been checking this. It seems we need a monkey patch to render liquid templates in the dev. portal. The upgrade broke the implementation and the patch needed to be adapted. The new implementation is easy, when it's rendering a liquid template, use the liquid renderer, otherwise use the default Rails renderer.

I'd say there must be a way to fix it without the monkey patch, but that's out of scope here, I think.

See https://github.com/3scale/porta/commit/775bbc4fb4da3f7d2a410cacb17bcdc0d1af23fd and https://github.com/3scale/porta/commit/540100a053516afc44d87123c0137f9542056417

Reply:
Thanks for the clarification!